docs/deployment/composer.md

```diff
@@ -1,5 +1,7 @@
 Drupal VM is configured to use `composer create-project` to build a Drupal 8 codebase by default but supports building Drupal from a custom `composer.json` file as well.
 
+_Note that if you already have a project built using a `composer.json` file you should instead add [Drupal VM as a dependency](composer-dependency.md) of your project. The method described below will make it somewhat difficult to manage your `composer.json` together with Drupal VM as the file will be copied from `drupal_composer_path` into `drupal_composer_install_dir` as a secondary `composer.json`._
```

Comment:
Are you okay with this? Honestly I think we should default `drupal_composer_path` to `no` in favor of Drupal VM as a dependency. I think we've seen a few issues due to this particular setup being a bit weird.